### ISSUE TITLE
Having difficulty extracting text from headings, images and unordered lists

### ISSUE THREAD

**Issue by boxabirds** [adbar/trafilatura#63]
Hi all I presume this is an easy fix. I'm loading a local copy of this file:

https://lisn-tests.netlify.app/rich-content.html
loaded using `Path(…).read_text()`

using `bare_extraction`

I get this output that appears to suggest that trafilatura with these options is skipping over headers, bullets, and discarding images. What have I done wrong to make it behave like this? 
```

  "title": "This is heading 1",
  "author": null,
  "hostname": null,
  "date": null,
  "categories": "",
  "tags": "",
  "fingerprint": "bnP8wg6PD0dg9QA9O/mFqg7dSEk=",
  "id": null,
  "raw-text": "Paragraph 1. What does this make of a bulleted list? This is quoted textAn image: This is bolded text. This is bolded text. This is a paragraph with some bold text in the middle. Table heading 1 Table heading 2 Table heading 1 Table heading 2 Table heading 1 Table heading 2",
  "source": null,
  "source-hostname": null,
  "excerpt": null,
  "text": "Paragraph 1. What does this make of a bulleted list?\nThis is quoted textAn image:\nThis is bolded text.\nThis is bolded text.\nThis is a paragraph with some bold text in the middle.\n|Table heading 1||Table heading 2|\n|Table heading 1||Table heading 2|\n|Table heading 1||Table heading 2|",
  "comments": ""
}
===
[
  "Paragraph 1.",
  "What does this make of a bulleted list?",
  "This is quoted textAn image:",
  "This is bolded text.",
  "This is bolded text.",
  "This is a paragraph with some bold text in the middle.",
  "|Table heading 1||Table heading 2|",
  "|Table heading 1||Table heading 2|",
  "|Table heading 1||Table heading 2|"
]
```

[Test program](https://gist.github.com/boxabirds/88fbf432b756e8def8aa49a36ec4e3ba)

**Comment by adbar** [comment]
Hi @boxabirds, this forged example isn't quite real, the software is designed to work on full-fledged web pages. There are several things happening here:
- trafilatura discards images by default, you'd have to set `include_images=True`
- in the absence of identifiable content containers it tries to gather certain text elements (which doesn't include headers or lists since it would impact precision)

**Comment by adbar** [comment]
I just noticed you did include images, this is related to #52: I plan to let advanced users decide which tags they want to keep.

**Comment by boxabirds** [comment]
Thanks @adbar  your work is amazing — yes I subsequently figured out that I needed to include in a very least an `article` tag. I assumed this package was a webpage content extraction tool. And I think it’s a great idea, as you suggest to allow people to tune which tags they consider articles. 

The background is that I’m migrating from `newspaper` with the test cases I constructed for that, and its approach isn’t as strict about article definitions.
